### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34169]
#### VFS team name
Debt Resolution

#### VFS product name
Medical Copays (MCP)

#### Point of Contact/Reviewers
Josh Kim (@joshkimux) - Accessibility

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
"Notice of rights and responsibilities" is a button styled as a link.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Style this button as a button as it is triggering a modal to open. Alternatively, you could write this as a link that navigates to a page about notice of rights and responsibilities instead and retain the link styling. Check out this presentation on buttons vs links for more on material honesty.

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by denisecoveyduc** [comment]
@rileyorr Can you take a look at this one?

**Comment by rileyorr** [comment]
@joshkimux thank you for the feedback! We referenced the 'Privacy Act Statement' modal button styling that's present on form intro pages such as [the 10-10EZ](https://staging.va.gov/health-care/apply/application/introduction) and [application for VA education benefits](https://staging.va.gov/education/apply-for-benefits-form-22-1990/introduction). I wanted to bring those instances to your attention in case those forms need to be updated to button styling as well! 

Through research, we learned that Veterans often don't read the 'Notice of rights and responsibilities' associated with their statements. However, our stakeholders requested we provide that content within the tool, so we made that content less prominent in the detail page's hierarchy. Considering how we expect Veterans to interact with this content, I think directing the user away from the detail page to a separate 'Notice of rights and responsibilities' page may hinder the experience. 

I've updated our designs so the link is now a secondary button. I'll also add a ticket to my team's backlog for the UI to be updated as well. Let me know if you have any questions or concerns!

**Comment by joshkimux** [comment]
<img width="712" alt="Screen Shot 2022-01-12 at 11 27 58 AM" src="https://user-images.githubusercontent.com/14154792/149224055-473661b6-a2ef-43ef-955f-6896348eb21b.png">

@rileyorr wonderful! Thank you for bringing those instances to my attention as well. I'll look into writing up tickets to address them as well 🙏

**Comment by joshkimux** [comment]
[Started a discussion on DSVA slack](https://dsva.slack.com/archives/C7NJWUA8K/p1642603442017600)

**Comment by caw310** [comment]
Per DSC meeting, put privacy notice in "Additional info".
